iceoryx_utils/include/iceoryx_utils/posix_wrapper/timer.hpp

```diff
@@ -321,6 +321,55 @@ class Timer
 };
 
 } // namespace posix
+
+/// @brief Class which offers the deadline timer functionality
+/// @note This class is introduced as an alternate timer functionality for the deprecated POSIX timers
+///
+/// @code
+///     iox::clock::DeadlineTimer dealineTimer(1000_ms);
+///
+///     // to check if the timer is active
+///     if( deadlineTimer.hasExpired()){
+///     ...
+///     }
+///     // to reset the timer and start
+///     TiborTheTimer.reset();
```

Comment:
This should match the variable above right ?
i.e. `deadlineTimer.reset();`

Reply:
Yes somehow the name was not saved i missed it. will fix this.

Reply:
Well, I like `TiborTheTimer` :)